lib/KIcon/precompiled-icons/le/a11y.vue

```diff
@@ -7,6 +7,6 @@
 
 <script>
 
-  export default {"name":"icon-40bcababd163c5d974599a8102319c63"}
+  export default {"name":"icon-90cbb33ba1b625a32a6da324f4db40b2"}
```

Comment:
We should not see changes to the files in `precompiled-icons/le` folder. Idk why this happened. We would need to revert them back. The only change that should be present is `lib/KIcon/precompiled-icons/le/codeToggle.vue`. Could you revert the commit and try adding the icon again and see if the script generates the same results?